desktop/buildSrc/src/main/kotlin/io/spine/internal/dependency/Material3.kt

```diff
@@ -30,7 +30,7 @@ package io.spine.internal.dependency
 public object Material3 {
 
     public object Desktop {
-        private const val version = "1.6.2"
+        private const val version = "1.7.0"
```

Comment:
I would move this object to `Compose` instead. Given these versions are in fact the same version, let's put the structure of our dependencies into accordance with that.